### PR TITLE
Update Nix docs

### PR DESCRIPTION
**Pull request by Scrumplex**
Parent PR: #1434

### CODE REVIEW

File: nix/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Prism Launcher Nix packaging
```
Comment (getchoo) [inline nix/README.md]:
```suggestion
# Prism Launcher Nix Packaging
```
title case :p

File: nix/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+Example:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    prismlauncher = {
+      url = "github:PrismLauncher/PrismLauncher";
+      # Optional: Override the nixpkgs input of prismlauncher to use the same revision as the rest of your flake
```
Comment (getchoo) [inline nix/README.md:43]: this should probably mention that overriding nixpkgs can break reproducibility with our flake

File: nix/README.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    };
+  };
+
+  outputs = {nixpkgs, prismlauncher}: {
+    nixosConfigurations.foo = nixpkgs.lib.nixosSystem {
+      system = "x86_64-linux";
+
+      modules = [
+        ({pkgs, ...}: {
+          nixpkgs.overlays = [prismlauncher.overlays.default];
```
Comment (getchoo) [inline nix/README.md:55]: ditto

File: nix/README.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    };
+  };
+
+  outputs = {nixpkgs, prismlauncher}: {
+    nixosConfigurations.foo = nixpkgs.lib.nixosSystem {
+      system = "x86_64-linux";
+
+      modules = [
+        ({pkgs, ...}: {
+          environment.systemPackages = [prismlauncher.packages.${pkgs.system}.prismlauncher];
```
Comment (getchoo) [inline nix/README.md:90]: this is perfectly reproducible assuming that nixpkgs hasn't been overridden. might be worth a mention

File: nix/README.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+You can add this repository as a channel and install its packages that way.
+
+Example:
+
+```shell
+nix-channel --add https://github.com/PrismLauncher/PrismLauncher/archive/develop.tar.gz prismlauncher
+
+nix-channel --update prismlauncher
+
+nix-env -iA prismlauncher
```
Comment (getchoo) [inline nix/README.md]:
```suggestion
nix-env -iA prismlauncher.prismlauncher
```
i'm pretty sure this is required with `nix-env` when using the `-A` flag

**Review by getchoo — CHANGES_REQUESTED** [review]
only other thing i suggest is mentioning how builds are cached by garnix, and how to add it to `substituters` with
```nix
{
  nix.settings = {
    trusted-substituters = [
      "https://cache.garnix.io"
    ];

    trusted-public-keys = [
      "cache.garnix.io:CTFPyKSLcx5RMJKfLo5EEPUObbA78b0YQ2DTCJXqr9g="
    ];
  };
}
```


testing pr builds with `nix run` or in a shell with `github:PrismLauncher/PrismLauncher?ref=pull/<PR_ID>/HEAD` also might be a cool mention

**Review by getchoo — APPROVED** [review]
lgtm :)

**Review by Trial97 — APPROVED** [review]

**Pull request merged**